src/beast/site_scons/site_tools/VSProject.py

```diff
@@ -583,6 +586,43 @@ def _guid(seed, name = None):
     guid = "{%s-%s-%s-%s-%s}" % (d[:8], d[8:12], d[12:16], d[16:20], d[20:32])
     return guid
 
+def _rmExeMod(file_name):
```

Comment:
1. Python is lucky as a language inasmuch it has One True Style that comes with the language, PEP8.  Accordingly, this function should use underscores to separate words, not camelCase.  I do understand, however, that this file wasn't written right.

2. This name is not so easy to understand!  What about `turn_off_executable_flag` or something?